dicesGame/src/main/java/com/example/dices/models/JsonModel.java

```diff
@@ -0,0 +1,28 @@
+package com.example.dices.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class JsonModel {
+
+    public int id;
+
+    public int roll;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public void setRoll(int roll) {
+        this.roll = roll;
+    }
```

Comment:
Please remove these setter methods. Pass the values in the constructor method instead.

Reply:
This lines were deleted.